diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts b/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
index patched
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
@@ -14,6 +14,9 @@
 }
 
 repositories {
+  maven { url = uri("https://maven.aliyun.com/repository/gradle-plugin") }
+  maven { url = uri("https://maven.aliyun.com/repository/public") }
+  maven { url = uri("https://maven.aliyun.com/repository/google") }
   google()
   mavenCentral()
 }
